tests/integration/pilot/istioctl_test.go

```diff
@@ -56,36 +55,6 @@ var (
 	describePodAOutput = describeSvcAOutput
 )
 
-func TestWait(t *testing.T) {
-	t.Skip("https://github.com/istio/istio/issues/29315")
```

Comment:
This issue is not fixed, so seems we should keep it skipped? Otherwise we will see flakes again

Reply:
If we push xds successfully ( no matter whether envoy recerive or load ), such that wait is true.

Reply:
the entire point of wait is that envoy has full received the config and has it processing currently. Just "it was pushed" doesn't provide much use

Reply:
I know, can you take a look at this commit https://github.com/istio/istio/pull/48542/commits/89697ae04979f1cb4701d74f940d9e7b6ca6740f

Reply:
You commented to not change the state record in processRequest, i reverted it